contrib/cirrus/setup.sh

```diff
@@ -12,23 +12,17 @@ cd $GOSRC
 msg "Setting up $OS_RELEASE_ID $OS_RELEASE_VER"
 case "$OS_RELEASE_ID" in
     fedora)
-        $LONG_DNFY update  # install latest packages
-        [[ -z "$RPMS_REQUIRED" ]] || \
-            $SHORT_DNFY install $RPMS_REQUIRED
+        # Required on Fedora VM images
+        bash "$SCRIPT_BASE/add_second_partition.sh"
         [[ -z "$RPMS_CONFLICTING" ]] || \
             $SHORT_DNFY erase $RPMS_CONFLICTING
-        # Only works on Fedora VM images
-        bash "$SCRIPT_BASE/add_second_partition.sh"
+        $SHORT_DNFY install zstd
         ;;
     ubuntu)
-        $SHORT_APTGET update  # Fetch latest package metadata
-        [[ -z "$DEBS_HOLD" ]] || \
-            apt-mark hold $DEBS_HOLD
-        $LONG_APTGET upgrade # install latest packages
-        [[ -z "$DEBS_REQUIRED" ]] || \
-            $SHORT_APTGET -q install $DEBS_REQUIRED
         [[ -z "$DEBS_CONFLICTING" ]] || \
             $SHORT_APTGET -q remove $DEBS_CONFLICTING
+        $SHORT_APTGET -q update
+        $SHORT_APTGET -q install zstd
```

Comment:
I think installing `zstd` at runtime is not necessary if you tell `.cirrus.yml` to use images with ID `c6374915067609088` (F32, F33 + U2004, U2010).

Reply:
Ya, [just update this line.](https://github.com/containers/storage/blob/ff9149bcd8a6d2419ba7811fe9477d664c31a55f/.cirrus.yml#L28)

Runtime package installs nearly always end badly...eventually :D

Reply:
Right, this is a backport from a PR that didn't want to have to wait for that.

Reply:
Oh okay, if it's a backport that's fine.  I'm okay either way then.